md_api/src/main/java/com/wipro/ats/bdre/md/api/base/MetadataAPIBase.java

```diff
@@ -22,6 +25,15 @@
  */
 public abstract class MetadataAPIBase extends BaseStructure {
 
+    private static AutowireCapableBeanFactory acbFactory = null;
+    protected static AutowireCapableBeanFactory getAutowireCapableBeanFactory(){
+        if(acbFactory==null) {
+            ApplicationContext context = new ClassPathXmlApplicationContext("spring-dao.xml");
+            acbFactory = context.getAutowireCapableBeanFactory();
+        }
+        return acbFactory;
+    }
+
 /*
     private static SqlSessionFactory sqlSessionFactory;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)